src/Quotebot/Bot.cs

```diff
@@ -50,18 +49,16 @@ public async Task Connect()
         await _client.StartAsync();
     }
 
-    public async Task LogMessage(LogMessage log)
+    private async Task LogMessage(LogMessage log)
     {
         _logger.LogDebug(log.Message);
         await Task.CompletedTask;
     }
 
-    public async Task OnReady()
+    private async Task OnReady()
     {
-        await _interactionService.RegisterCommandsToGuildAsync(_configuration.GuildId, true);
```

Comment:
I'm curious what parameter we were setting to true here. Might be off-topic, but I'm a fan of naming parameters for better readability.